client/client.go

```diff
@@ -660,7 +660,7 @@ func (c *client) Close() {
 	}
 }
 
-func (c *client) setServiceMode(newMode pdpb.ServiceMode) {
+func (c *client) setServiceMode(newMode pdpb.ServiceMode, skipSameMode bool) {
```

Comment:
Prefer using a more straightforward word.

```suggestion
func (c *client) setServiceMode(newMode pdpb.ServiceMode, force bool) {
```

Reply:
It's not the same as force.

Reply:
why not skipSameMode always?